response_operations_ui/forms.py

```diff
@@ -105,10 +105,10 @@ class UpdateEventDateForm(FlaskForm):
 
     HOURS = [(hour, hour) for hour in ['00', '01', '02', '03', '04', '05', '06', '07', '08', '09', '10', '11',
                                        '12', '13', '14', '15', '16', '17', '18', '19', '20', '21', '22', '23']]
-    hour = SelectField('hours', choices=HOURS)
+    hour = SelectField('hours', choices=HOURS, default='07')
 
     MINUTES = [('00', '00'), ('15', '15'), ('30', '30'), ('45', '45')]
-    minute = SelectField('minutes', choices=MINUTES)
+    minute = SelectField('minutes', choices=MINUTES, default='00')
     submit = SubmitField('Save')
 
     def validate_day(form, field):
```

Comment:
I know its not part of your PR but its a quick fix and could be quickly done now since your editing the file now? Whenever validating certain attributes for the CE's we're passing in the unused param `form` often would be worth removing i.e. `validate_period`  etc.
https://github.com/ONSdigital/response-operations-ui/pull/190/files#diff-8e743c208190a309d86fe86016048666R127